test/unit/package_item_test.rb

```diff
@@ -0,0 +1,237 @@
+require 'test_helper'
+
+class PackageItemTest < Minitest::Test
+  def setup
+    @name = "Fancy Pants"
+    @weight = 100
+    @value = 1299
+    @quantity = 3
+    @sku = "1234567890"
+    @hs_code = "1234.56.78"
+    @options = {
+      units: :metric,
+      sku: @sku,
+      hs_code: @hs_code,
+      type: :actual,
+    }
+
+    @item = PackageItem.new(@name, @weight, @value, @quantity, @options)
+    @mass = ::Quantified::Mass.new(@weight, :grams)
+  end
+
+  def test_initialize_with_all_attributes
+    assert_equal @name, @item.name
+    assert_equal @options, @item.options
+    assert_equal @hs_code, @item.hs_code
+    assert_equal @sku, @item.sku
+    assert_equal @value, @item.value
+    assert_equal @quantity, @item.quantity
+  end
+
+  def test_initialize_assumes_symbol_keys
+    options = {
+      "units" => :imperial,
+      "sku" => @sku,
+      "hs_code" => @hs_code,
+    }
+    @item = PackageItem.new(@name, @weight, @value, @quantity, options)
+
+    assert_nil @item.hs_code
+    assert_nil @item.sku
+    refute_equal @weight, @item.ounces
+    assert_equal @weight, @item.grams
+  end
+
+  def test_initialize_negative_quantity
```

Comment:
I really dislike the `def test_` DSL here. It's part of what makes minitest so hard to work with.

I'll be coming back and naming all the tests with `test "#method_name action conditions" do` as part of 2.0. So I'm not really worried about the naming here.